pyignite/connection/connection.py

```diff
@@ -227,7 +213,7 @@ def _connect_version(self) -> Union[dict, OrderedDict]:
             self.send(stream.getbuffer(), reconnect=False)
 
         with BinaryStream(self.client, self.recv(reconnect=False)) as stream:
-            hs_response = HandshakeResponse.parse(stream, self.protocol_version)
+            hs_response = HandshakeResponse.parse(stream, self.protocol_context)
```

Comment:
We should update from response `protocol_context.features`

Reply:
I update it a bit later